source/contributing/howtogit.rst

```diff
@@ -0,0 +1,352 @@
+===============
+How to Git(Hub)
+===============
+
+If you want to assist in creating Sponge, you have an awesome addition to the API, or you want to improve our Docs,
+then you'll need to become familiar with ``git`` and GitHub. If you're already familiar with forking, branches,
+issues, pull-requests and commits, then just skip this topic. If you have no clue what we're talking about, then read on.
+
+.. note::
+  This guide assumes that you've read :doc:`../preparing/git` and that you've already setup your machine with a Git
+  client of your choice.
+
+The Basic Concept of Git and GitHub
```

Comment:
Thanks for the suggetion. I'll remove my glossary and just link to the one provided by GitHub.